src/api/handlers/guild.ts

```diff
@@ -304,6 +305,52 @@ export function emojiURL(id: string, animated = false) {
   return `https://cdn.discordapp.com/emojis/${id}.${animated ? "gif" : "png"}`;
 }
 
+/**
+ * ⚠️ **If you need this, you are probably doing something wrong. Always use cache.guilds.get()?.emojis
+ * 
+ * Returns a list of emojis for the given guild.
+ */
+export async function getEmojis(guildID: string, addToCache = false) {
+  const result = await RequestManager.get(
+    endpoints.GUILD_EMOJIS(guildID),
+  ) as Emoji[];
+
+  if (addToCache) {
+    const guild = await cacheHandlers.get("guilds", guildID);
+    if (!guild) throw new Error(Errors.GUILD_NOT_FOUND);
+    cacheHandlers.set("guilds", guildID, { ...guild, emojis: result });
+  }
+
+  return result;
+}
+
+/**
+ * ⚠️ **If you need this, you are probably doing something wrong. Always use cache.guilds.get()?.emojis
```

Comment:
Move this at the end of the comment block, the description of the function should come first.